suite-native/device-manager/src/components/DeviceManagerContent.tsx

```diff
@@ -45,26 +46,27 @@ export const DeviceManagerContent = () => {
         });
     };
 
+    const listedDevice = devices.filter(device => device.id !== selectedDeviceId);
```

Comment:
useMemo

Reply:
memo applied in [4092740](https://github.com/trezor/trezor-suite/pull/10020/commits/4092740700d31090573c5fbf379279c641a91726)